### DISCUSSION TITLE
Module not found: Default condition should be last one

### DISCUSSION THREAD

**Discussion by haeli05** [vercel/community#2242] [Help] [answered]
### Summary

I'm getting a bug:
Module not found: Default condition should be last one

Which I can fix locally.

The bug fix simply involves going to node_modules/alchemy-sdk/package.json and moving the "default" export to the last in the line of objects.

On Vercel I can't do this, obviously.

I've made a bug report to alchemy-sdk but until there's a fix, how do I address this issue?

### Additional information

The bug fix simply involves going to node_modules/alchemy-sdk/package.json and moving the "default" export to the last in the line of objects.

**Comment by amyegan** [comment, marked as answer]
Hi @haeli05! It looks like the problem was introduced a couple of weeks ago in https://github.com/alchemyplatform/alchemy-sdk-js/pull/303. I suspect you could go back one version to [2.8.0](https://www.npmjs.com/package/alchemy-sdk/v/2.8.0) as a temporary workaround. You'll just need to remove the `^` or `~` so the minor or patch version doesn't change until a more permanent fix is available
